examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -0,0 +1,281 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+class ExpasionPanelsDemo extends StatefulWidget {
+  static const String routeName = '/expansion_panels';
+
+  @override
+  _ExpansionPanelDemoState createState() => new _ExpansionPanelDemoState();
+}
+
+enum _Location {
+  Barbados,
+  Bahamas,
+  Bermuda
+}
+
+typedef String StringValue();
+
+class _ExpansionPanelDemoState extends State<ExpasionPanelsDemo> {
```

Comment:
The convention is to give this class the same name as the StatefulWidget plus "State", so _ExpansionPanelsDemoState.